packages/db/src/index.ts

```diff
@@ -1,11 +1,264 @@
+ * is implemented and even as the network to which it's deployed experiences
+ * a hard-fork. **Blockchain data is not forgotten — neither should blockchain
+ * metadata.**
+ *
+ * At a high-level, this package provides a
+ * [GraphQL](https://graphql.org/) interface and stores data via one of
+ * several persistence backends thanks to [PouchDB](https://pouchdb.com/).
+ * Use of this package directly is intended mostly for other tools – end users
+ * can find interfaces to @truffle/db by way of Truffle itself, e.g. with the
+ * `truffle db serve` command that starts a GraphQL Playground HTTP server.
+ *
+ * This documentation serves to organize the modules and namespaces included
+ * in the package, both for tool-developer reference and for continued work
+ * on @truffle/db itself. (Disclaimer: as a result, this API documentation
+ * may serve neither of these purposes well. Please reach out with questions
+ * and/or to suggest helpful clarifications!)
+ *
+ * Continue reading below for an overview and full index of this package's
+ * exports.
+ *
+ * ---
+ *
+ * <figure style="text-align: center">
+ * <img
+ *   src="media://images/example-query.png"
+ *   alt="Example query"
+ *   style="width: 85%" />
+ * <figcaption style="font-size: small; font-style: italic">
+ * <strong>Figure</strong>:
+ * Example query for a project's <code>MagicSquare</code> contract
+ * </figcaption>
+ * </figure>
+ *
+ * ---
+ *
+ * ## Contents
+ *
+ * For quick reference, this documentation summary contains the following
+ * headings:
+ *
+ * - [Introduction](#introduction)
+ *
+ * - [Contents](#contents)
+ *
+ * - [Core library interface](#core-library-interface)
+ *   - [Instantiating @truffle/db](#instantiating-truffledb)
+ *   - [GraphQL schema](#graphql-schema)
+ *
+ * - [Data model](#data-model)
+ *   - [Structure](#structure)
+ *   - [List of collections](#list-of-collections)
+ *
+ * - [Other interfaces](#other-interfaces)
+ *   - [JavaScript / TypeScript interface](#javascript---typescript-interface)
+ *   - [Truffle-specific interface](#truffle-specific-interface)
+ *   - [HTTP interface](#http-interface)
+ *
+ * - [Additional materials](#additional-materials)
+ *
+ * ## Core library interface
+ *
+ * ### Instantiating @truffle/db
+ *
+ * This package defines the primary [[connect | `connect()`]] function, which
+ * returns an object adhering to the [[Db]] interface for given
+ * [ConnectOptions](#connectoptions).
+ * This [[Db]] interface defines the `async`
+ * [[Meta.Db.execute | `db.execute()`]]
+ * method that accepts a GraphQL request and returns a GraphQL response.
+ *
+ * ### GraphQL schema
+ *
+ * \@truffle/db makes its GraphQL schema available as the exported
+ * [[Graph.schema | `Graph.schema`]] variable, or view the SDL
+ * details in the [[Graph]] namespace description.
+ *
+ * ## Data model
+ *
+ * ### Structure
+ *
+ * Data is organized as a number of collections of representations of key
```

Comment:
```suggestion
 * Data is organized as collections of representations of key
```

Reply:
Hm having trouble putting my finger on the distinction I was trying to make by framing it this way... I want to indicate that "collections" is a well-defined thing. I think you're right, though, that it's evident enough thanks to context.

Good suggestion!